metadata.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.16.0 < 10.0.0"
+      "version_requirement": ">= 9.0.0 < 10.0.0"
```

Comment:
Changing the lower version *is* a breaking change (e.g. users of version 7.0.0 will have unmet dependencies when updating, and ignoring it would break their catalog as the namespaced function is not available).